README.md

```diff
@@ -1,3 +1,6 @@
-# Hello World
-This is my first attempt at using a GIT Repository.
-This is the first change.
\ No newline at end of file
+# Hello World
+This is my first attempt at using a GIT Repository.
+
```

Comment:
Did you change this to UNIX line endings?

Reply:
Yep!

Reply:
Noice